ABOUT.org

```diff
@@ -0,0 +1,285 @@
+* kixi.datastore
+
+**** May 2018
+
+/This document assumes familiarity with the Witan architecture, CQRS and event-sourcing./
+
+*kixi.datastore* (aka 'The Datastore') is the service concerned with /files/ and /metadata/. These two primitives form the basis for all of the Datastore's interactions.
+
+To clarify the abstractions, in the Datastore /files/ refer to the literal kind of computer file. A blob of specifically-formatted binary data, interpretable
+by any application which understands the format. One feature of the Datastore is to store these kinds of files, so that they can be distributed, downloaded
+or otherwise accessed in the pursuits of data science. Files, as they exist on a computer, do not have a standardised mechanism for transmitting metadata
+and therefore the Datastore provides this mechanism - known in the system simple as /metadata/. A /file/ always has an accompanying /metadata/. This /metadata/
+describes a multitude of things, from the file's name and time it was uploaded, to whom in the system is allowed to access it.
+
+Whilst it's true to say that a /file/ always has a /metadata/ (and in fact they share a UUID), the opposite is not true. The concept of /metadata/ has been expanded (and there are plans to
+expand it further) to include "/bundles/" - collections of other /metadata/. One such implementation of a /bundle/ is a /datapack/.
+
+** History
+
+The original concept for the Datastore was as a supplementary system, to simply facilitate getting files into a location from where we could run them
+through models or "workspaces". To this end there were some ambitious features, the relics of which are still present in the codebase - file segmentation,
```

Comment:
can you point to any of the namespaces that cover these? They'd be good for review and removal.